avalon/tools/contextmanager/app.py

```diff
@@ -20,6 +21,7 @@ class App(QtWidgets.QDialog):
     def __init__(self, parent=None):
         QtWidgets.QDialog.__init__(self, parent)
 
+        self.setAttribute(QtCore.Qt.WA_DeleteOnClose, True)
```

Comment:
If this is true, then the instance we store in `module.window` will be invalidated. It is deliberately *not* deleted such that it can be reused later. The window is preserved to also preserve e.g. user selection and focus within the app. I think we want those things?

Reply:
Good point. Will remove that.

Reply:
Actually looking closer, it seems like we are deleting the old window anyways: https://github.com/getavalon/core/blob/master/avalon/tools/contextmanager/app.py#L218-L222

Reply:
Currently only Loader's window get preserved, and I vote for keeping them :P